harness/tests/determined/common/experimental/test_determined.py

```diff
@@ -141,3 +142,71 @@ def test_get_trial_populates_attribute(
     resp = client.get_trial(trial_id=trial_id)
 
     assert attribute in resp.__dict__
+
+
+@responses.activate
+@mock.patch("determined.common.api.bindings.get_GetExperiments")
+def test_list_experiments_calls_bindings_with_params(
```

Comment:
a bit handwavy, but i figured there should be a test for any logic in `list_experiments` before bindings/HTTP layer since that's the only behavior `list_experiments` is really responsible for. if we add logic to `list_experiments` for more complicated filtering, etc, this test might be useful.